docs/source/explicit_comms.rst

```diff
@@ -0,0 +1,16 @@
+Explicit-comms
+==============
+
+Communication and scheduling overhead can be a major bottleneck in Dask/Distributed. Dask-CUDA addresses this by introducing an API for explicit communication in Dask tasks.
+The idea is that Dask/Distributed spawns workers and distribute data as usually while the user can submit tasks on the workers that communicate explicitly.
```

Comment:
Maybe we should add what kind of operations/functions can benefit from explicit comms?

Reply:
Added some more text